tests/test_github_pagination.py

```diff
@@ -21,12 +21,30 @@ def get_properties(self, original: bool = True):
         return return_value
 
     def test_run(self):
+        # For some streams RECORD count were not > 30 in same test-repo. So, separated streams on the basis of RECORD count 
+        # Pagination is not supported for "team_memberships" by Github API
+        # Skipping "teams" stream as it's RECORD count is <= 30
+        self.repository_name = 'singer-io/tap-github'
+        expected_stream_1 = {'comments', 'stargazers', 'commits', 'pull_requests', 'reviews', 'review_comments', 'pr_commits', 'issues'} 
+        self.run_test(expected_stream_1)
+        
+        self.repository_name = 'singer-io/test-repo'
+        expected_stream_2 = {'issue_labels', 'events', 'collaborators', 'issue_events', 'team_members', 'assignees', 'commit_comments', 'projects', 'project_cards', 'project_columns', 'issue_milestones', 'releases'}
```

Comment:
Define `expected_streams = self.expected_streams()`  and `untestable_streams`, then choose `expected_stream_2 = expected_streams - expected_stream_1 - untestable_streams` so it will make sure that no streams are left in test consideration.

Reply:
Updated